src/templates/components/audienceTopics/index.tsx

```diff
@@ -6,10 +6,11 @@ export function AudienceTopics({ tags }: FormattedAudienceTopics) {
   if (isEmpty(tags)) return null
   const tagsList = tags.map(({ id, href, name }) => (
     <div key={id}>
-      <div className="vads-u-margin-right--1 vads-u-margin-bottom--1 medium-screen:vads-u-margin-bottom--0">
+      <div className="vads-u-margin-right--1 vads-u-margin-bottom--1 medium-screen:vads-u-margin-bottom--1p5">
         <Link
           href={`${href}/${encodeURI(name)}`}
```

Comment:
i don't think this `href` is correct. the `/${encodeURI(name)}` portion can be dropped.

looking at va.gov: 
![Screenshot 2024-01-30 at 8 29 26 AM](https://github.com/department-of-veterans-affairs/next-build/assets/11279744/316d8c3c-5069-4c1e-ab1c-274d69e679f4)


this pr:
`/resources/tag/all-veterans/All%20Veterans`

Reply:
good catch, ty